packages/flutter/test/material/theme_test.dart

```diff
@@ -298,6 +298,55 @@ void main() {
     expect(glyphText.text.style.fontSize, 20.0);
   });
 
+  testWidgets('default hint color passes contract test - light', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const MaterialApp(
+        home: Scaffold(
+          body: TextField(
+            decoration: InputDecoration(
+              labelText: 'label text',
+            ),
+          ),
+        ),
+      ),
+    );
+    await expectLater(
+      tester,
+      meetsGuideline(
+        CustomMinimumContrastGuideline(
+          finder: find.text('label text'),
+          minimumRatio: 5.5,
```

Comment:
I picked a value that is very close the to actual contract(5.6) which will fail without the change. Not sure if we have a constant define somewhere?

Reply:
I think we should MinimumTextContrastGuideline here, that's what we use in most text contrast tests.

Reply:
I did try using that, but the label text pass the MinimumTextContrastGuideline even before the change. Do we still want to use it for this test?

Reply:
The original issue was a complaint about a11y text contrast:

> It seems like for label text we are using grey[600] which fails the minimum contrast test with the white background. It seems that it needs to be at least grey[700]. I could not find Material specification for this.

@mehmetf - is there a  regression test that currently fails because of the label text colors?

Reply:
IIUC, the failure is coming from manual a11y scanner testing; not an automated regression test.

See: b/154474989. I am not sure if there's an automated equivalent of scanner. I am not aware of one.

Reply:
I am kinda new to color mixing, but how would you have a 80868B if the background is FFFFFFFF and the color is 80000000, I will expect the (R,B,G) to all have the same number.

Reply:
Does the app has its own theme with custom hintColor?